src/main/java/com/jensjansson/ce/views/persons/EditorView.java

```diff
@@ -195,6 +196,14 @@ protected void editPerson(Person person) {
         input.setTopic(topicId);
         if (topicId != null) {
             BotRunner.onUserJoined(topicId, localUser, person, personService);
+            CollaborationEngine.getInstance().openTopicConnection(this, topicId,
+                    localUser, topicConnection -> {
+                        topicConnection
+                                .getNamedMap(CollaborationMessageList.class
+                                        .getName())
+                                .setExpirationTimeout(Duration.ofMinutes(15));
```

Comment:
This should be `getNamedList`, but unfortunately `CollaborationList` doesn't have the expiration timeout API yet: https://github.com/vaadin/collaboration-engine-internal/issues/501

Reply:
The version of CE that was available at the time did still use maps. But also, it did no actually expire the messages. I'm abandoning this PR now. We can add it back when we support expiration timeout for lists.